src/styles/reset.css

```diff
@@ -0,0 +1,12 @@
+body,
+div,
+p,
+h1,
+h2,
+h3,
+h4,
+a,
+ul {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
only reset styles for elements you use in the project. also you don't need both normalize and reset files, use either

Reply:
Thank you for reviewing! Do you mean to move styles from reset.css to normalize.css and delete reset.css?

Reply:
> only reset styles for elements you use in the project. also you don't need both normalize and reset files, use either

.

Reply:
> Thank you for reviewing! Do you mean to move styles from reset.css to normalize.css and delete reset.css?

correct